index.html

```diff
@@ -1,279 +1,274 @@
+                                </li>
+                                <li><p class="first_project_holder1">Ruby</p>
+                                </li>
+                            </ul>
+                             <button class="see_project_button2 see_project_button2a">See Project</button>
+                        </div>
+                    
+                </div>
+            </div>
+        </div>
     </section>
-    <section class="third_section_container">
-      <div class="about_us_div">
-          <h2 class="about_title">About me</h2>
-          <p class="about_content">
-            Hello I’m a software developer! I can help you build a product ,
-            feature or website Look through some of my work and experience! If
-            you like what you see and have a project you need coded, don’t
-            hestiate to contact me.
-          </p>
-           <div class="resume_button">
-       <a class="button_resume"> Get My Resume </a>
-      </div>
-             </div>
-        <div class="line_divider_container">
-          <hr class="line_divider2" />
+     <section class="About_me_section">
+        <div class="About_me_div">
+            <div class="About_me_content">
+                <h2 class="About_me_title">About Me</h2>
+                <p class="About_me_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <button class="Resume_button">Get My Resume</button>
+            </div>
         </div>
-         <div class="image_frame_holder">
-              </div>
-        <div class="skill_display">
-          <ul class="skill_display_list">
-            <li class="skill_display_items1">
-              <div class="skill_display_lang">
-                <img class="skill_display_rectangle55" src="images/Rectangle 55.png" />
-                <p class="skill_display_title1">Language</p>
-              </div>
-              <ul class="skill_item_list1">
-                <li>Javascript</li>
-                <li>Ruby</li>
-                <li>Html</li>
-                <li>Css</li>
-              </ul>
-            </li>
-            <li class="skill_display_items2">
-              <div class="skill_display_framework">
-                <img class="skill_display_rectangle56" src="images/Rectangle 56.png" />
-                <p class="skill_display_title2">Frameworks</p>
-              </div>
-              <ul class="skill_item_list2">
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-              </ul>
-            </li>
-            <li class="skill_display_items3">
-              <div class="skill_display_Skills">
-                <img class="skill_display_rectangle57" src="images/Rectangle 57.png" />
```

Comment:
- [ ] Please stick to the Figma design as much as possible. kindly focus on padding and margin as in the Figma template. now your portfolio looks a bit messy.
## your design👇
![skilllsst](https://user-images.githubusercontent.com/66992901/194669159-d86c463e-10f1-4b72-820a-7a62f13e7871.PNG)
## Figma  design👇
![skillsfigma](https://user-images.githubusercontent.com/66992901/194669207-a1bd3ddb-0a40-45ed-91db-188fa6faa8c1.PNG)